webservices/common/models/reports.py

```diff
@@ -60,7 +60,7 @@ def treasurer_name(self):
         return name
 
 
-class CommitteeReports(PdfMixin, CsvFecMixin, BaseModel):
+class CommitteeReports(PdfMixin, CsvMixin, BaseModel):
```

Comment:
The fec url is now being added to the material view via a sql function, so the Mixin is not longer needed.